### ISSUE TITLE
Campaign progress lost for sector 80.

### ISSUE THREAD

**Issue by mdwerner68** [Anuken/Mindustry#10127]
### Platforms

Windows

### Build

146

### Issue

Twice now I have captured then almost fully built out sector 80 and working on it long enough to survive a second set of waves then logged out with 'save and quit' then quit the game.

After coming back to play some more I started the game, returned to Campaign mode and returned to finish up working the sector only to find everything gone but the core.  Gone, not destroyed.  And I am under attack again from the very start.

For some reason my game didn't save my progress correctly and, as you can imagine, this is very frustrating.  I am afraid to spend any more time working on this sector.  So far, I've only experienced this issue on sector 80.

I explored the data and discovered there exist two save files per sector and I copied them but their size seems small and I expect they are empty.  Curious if there is a method the replace the sector main save with the one denoted as backup, though I don't expect this would help restore my build.  I will experiment some more with smaller changes then saving and quit before restarting to see if things saved correctly.  If I get some partial progress saves I will manually copy them as backup along the way rather than doing all the work in one session.

### Steps to reproduce

I captured the sector, built it out extensively, survived a second set of waves, built it out some more to fully defend as well as produce the max of nearly every resource then quit the game properly with Esc>>Save and Quit>>Quit.

After returning to play some more I found all my progress gone with just a core and under attack back to wave 1.

After returning again I noticed that I now had zero resource (had 4.5k of each previously) and absolutely nothing but the core so I abandoned the sector.  I will experiment by launching on it again and building a few things then quitting to see if progress is saved for the sector.

### Mods used

None

### Save file

[Sector80Issue.zip](https://github.com/user-attachments/files/16646323/Sector80Issue.zip)

### (Crash) logs

NA

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.
- [X] I am not using Foo's Client, and have made sure the bug is not caused by mods I have installed.

**Comment by Anuken** [comment]
Unfortunately, there's not much I can do to help you. I've played the campaign(s) extensively and never had this issue; it's also not something that you can get logs of. People have experienced campaign save corruptions, yes, but not like what you're describing. There have also been changes to the way backups are handled in the development version, which may affect results. If multiple people report this problem after release, I'll look into it further.

**Comment by mdwerner68** [comment]
Thanks for the reply.  I have recaptured the sector but this time I saved and quit periodically as I made progress then copied the two save files for the sector to sequentially numbered sector subfolders (80-1, 80-2, 80-3 (Sector Captured), etc.) in a progress save folder.  My progress is being saved and I also have backups at various steps along the way.  I may do this for a while for all sectors just in case.  I'll report back if I experience this again.
